docs/general/ad-filtering/create-own-filters.md

```diff
@@ -267,6 +270,26 @@ In the following examples it is implied that requests are sent from `http://exam
 >
 > Safari does not support the simultaneous use of allowed and disallowed domains, so rules like `||baddomain.com^$domain=example.org|~foo.example.org` will not work in AdGuard for Safari.
 
+> Starting with CoreLibs version 1.12, the `$domain` modifier can be alternatively spelled as `$from`.
+
+#### **`$to`** {#to-modifier}
+
+`$to` limits the rule scope to requests made **to** the specified domains and their subdomains.
+To add multiple domains to one rule, use the `|`  character as a separator.
+
+**Examples**
+
+* `/ads$to=evil.com|evil.org` — block any request to `evil.com` or `evil.org` (and their subdomains) with a path matching `/ads`.
+* `/ads$to=~not.evil.com|evil.com` — block any request to `evil.com` (and its subdomains), with a path matching `/ads`,
+  except requests to `not.evil.com` (and its subdomains).
+* `/ads$to=~good.com|~good.org` — block any request with a path matching `/ads`,
+  except requests to `good.com` or `good.org` (and their subdomains).
+
+> `$to` is availabe starting with CoreLibs version **1.12**.
+
+> [`$denyallow`](#denyallow-modifier) can be expressed with inverted `$to`:
```

Comment:
for such one `Compatibility with other modifiers` is needed
and it should be placed before AdGuard version compatibility note